package.json

```diff
@@ -2,7 +2,11 @@
   "name": "yargs",
   "version": "12.0.2",
   "description": "yargs the modern, pirate-themed, successor to optimist.",
+  "author": "James Halliday",
   "main": "./index.js",
+  "contributors": [
+    {"name": "Yargs Contributors", "url": "https://yargs.js.org/"}
```

Comment:
rather thank linking to the website here, let's link to contributors on GitHub:

https://github.com/yargs/yargs/graphs/contributors

Reply:
OK, link updated!